src/main/java/edu/cornell/kfs/concur/rest/application/ConcurTestConnectionApplication.java

```diff
@@ -0,0 +1,31 @@
+package edu.cornell.kfs.concur.rest.application;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.core.Application;
+
+import edu.cornell.kfs.concur.service.impl.ConcurTestConnectionWebServiceImpl;
+
+@ApplicationPath("/systems/v1.0")
+public class ConcurTestConnectionApplication extends Application {
+
+    protected Set<Object> singletons = new HashSet<>();
+    protected Set<Class<?>> clazzes = new HashSet<>();
+
+    public ConcurTestConnectionApplication() {
+        singletons.add(new ConcurTestConnectionWebServiceImpl());
```

Comment:
The pattern in the base Financials code is that Resource classes are registered in the Application and the Resources calls services. I know this service isn't complex, but it would probably be best to follow that pattern here as well.